src/Functions/match.cpp

```diff
@@ -20,6 +20,7 @@ using FunctionMatch = FunctionsStringSearch<MatchImpl<NameMatch, false>>;
 void registerFunctionMatch(FunctionFactory & factory)
 {
     factory.registerFunction<FunctionMatch>();
+    factory.registerAlias("REGEXP_MATCHES", NameMatch::name, FunctionFactory::CaseInsensitive);
```

Comment:
This seems incorrect. Postgres's `regexp_matches` function doesn't return a boolean indicating whether a pattern matches a string, but returns a set of arrays of matches: https://www.postgresql.org/docs/current/functions-matching.html.